beacon_node/http_api/src/lib.rs

```diff
@@ -3875,7 +3875,7 @@ pub fn serve<T: BeaconChainTypes>(
         .and(chain_filter.clone())
         .then(
             |epoch: Epoch,
-             indices: Vec<u64>,
+             indices: api_types::ValidatorIndexData,
```

Comment:
I think we should also update the client in `common/eth2` so it sends quoted indices

Reply:
Just to make sure we're on the same page

The changes in `common/eth2` also require changes downstream to the doppleganger service, plus maybe a few other places

Also, reviewing the beacon api spec, the following endpoints should also receive quoted indices

`POST validator/duties/attester/{epoch}`
`POST validator/duties/sync/{epoch}`

will make changes in `common/eth2` for those as well if that sounds good

Reply:
I think those 2 endpoints are already doing the quoting via the `ValidatorIndexData{Ref}` types

https://github.com/sigp/lighthouse/blob/c7e5dd1098a145c0d2174dc65bd23faeb5074249/common/eth2/src/lib.rs#L2358-L2380

https://github.com/sigp/lighthouse/blob/c7e5dd1098a145c0d2174dc65bd23faeb5074249/common/eth2/src/types.rs#L693-L702

We can do similarly for this endpoint, where we pass in a `&[u64]` to the client function in `common/eth2`, and then send it quoted. This will unfortunately be a breaking change, but only for the liveness endpoint.